spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/adapter/BatchMessagingMessageListenerAdapter.java

```diff
@@ -63,8 +64,18 @@ public void onMessageBatch(List<org.springframework.amqp.core.Message> messages,
 		else {
 			List<Message<?>> messagingMessages = new ArrayList<>();
 			for (org.springframework.amqp.core.Message message : messages) {
-				messagingMessages.add(toMessagingMessage(message));
-			}
+                try {
+					Message<?> messagingMessage = toMessagingMessage(message);
+					messagingMessages.add(messagingMessage);
+                } catch (MessageConversionException e) {
+                    this.logger.trace("Could not convert incoming message", e);
```

Comment:
Thanks; this should work, but the log should be `error()`.

Also, please add a test.

Reply:
ok, It's all resolved